src/stability_sdk/animation.py

```diff
@@ -135,15 +124,23 @@ class VideoInputSettings(param.Parameterized):
     video_mix_in_curve = param.String(default="0:(0.02)")
     video_flow_warp = param.Boolean(default=True, doc="Whether or not to transfer the optical flow from the video to the generated animation as a warp effect.")
 
+class VideoOutputSettings(param.Parameterized):
+    fps = param.Integer(default=24, doc="Frame rate to use when generating video output.")
+    reverse = param.Boolean(default=False, doc="Whether to reverse the output video or not.")
+    vr_mode = param.Boolean(default=False, doc="Output views for both eyes in combination with 3D mode.")
```

Comment:
Looking at the implementation, I think this feature is also available to non-3D modes.. and in some situations (but admittedly, certain use of non-3D modes could be problematic I think), the results could be pretty interesting. I think it should be re-worded: Something to indicate that it outputs two images per frame corresponding to stereoscopic vision, leveraging depth estimation and 3D transformations... (although that's pretty long-winded for these descriptions.. so I see the difficulty here).

Reply:
Oh yes good point, that doc string is misleading as it should in fact work for any mode. I will test it out and update the doc string. 

I had also considered naming it stereoscopic mode instead of VR because the "VR" part with the projection to get 180 view is optional. For sweeping landscapes the spherical projection is nice but for cheeseburger rotation just doing regular stereoscopic side by side with no spherical projection is better. I settled on VR because it's shorter, is what it was named in Disco, and with the spherical projection is more than just stereoscopic rendering - oh and with no more stereoscopic TVs or monitors anymore it's really just in VR that you can watch these now I believe.